### ISSUE TITLE
fingerbank_data_update does nothing on standalone server

### ISSUE THREAD

**Issue by julsemaan** [inverse-inc/packetfence#5867]
**Describe the bug**
The fingerbank_data_update maintenance job does nothing when being run on a standalone server.

**Comment by julsemaan** [comment]
Cluster is broken though. It only updates the DB on the first server:

```
[root@cluster1 pf]# grep fingerbank logs/httpd.webservices.access 
Sep 30 09:56:36 cluster1 httpd_webservices: 127.0.0.1 - fingerbank_update_component [30/Sep/2020:09:56:36 -0400] "POST / HTTP/1.1" 200 208 320 30140 "-" "Go-http-client/1.1" "127.0.0.1:9090"
Sep 30 09:57:36 cluster1 httpd_webservices: 127.0.0.1 - fingerbank_update_component [30/Sep/2020:09:57:36 -0400] "POST / HTTP/1.1" 200 208 320 5837 "-" "Go-http-client/1.1" "127.0.0.1:9090"
Sep 30 09:58:36 cluster1 httpd_webservices: 127.0.0.1 - fingerbank_update_component [30/Sep/2020:09:58:36 -0400] "POST / HTTP/1.1" 200 208 320 6048 "-" "Go-http-client/1.1" "127.0.0.1:9090"
Sep 30 09:59:36 cluster1 httpd_webservices: 127.0.0.1 - fingerbank_update_component [30/Sep/2020:09:59:36 -0400] "POST / HTTP/1.1" 200 208 320 31011 "-" "Go-http-client/1.1" "127.0.0.1:9090"
Sep 30 10:00:36 cluster1 httpd_webservices: 127.0.0.1 - fingerbank_update_component [30/Sep/2020:10:00:36 -0400] "POST / HTTP/1.1" 200 208 320 27481 "-" "Go-http-client/1.1" "127.0.0.1:9090"
[root@cluster2 pf]# grep fingerbank logs/httpd.webservices.access
Nothing...
[root@cluster3 pf]# grep fingerbank logs/httpd.webservices.access
Nothing...
```

**Comment by jrouzierinverse** [comment]
This is fixed by 077f9d718c221a1d0e63c33b02db3249055c9143 and f4b78248acac6bd0d2b34375707fb02a81f787dc.
